benchmarks/pool_stress_test.cpp

```diff
@@ -0,0 +1,93 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+// project contributors. See the COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (MIT)
+//////////////////////////////////////////////////////////////////////////////
+#include <iostream>
+#include <chrono>
+#include <string>
+#include <random>
+#include <numeric>
+
+#include "umpire/config.hpp"
+#include "umpire/ResourceManager.hpp"
+#include "umpire/Allocator.hpp"
+#include "umpire/strategy/MixedPool.hpp"
+
+#define CONVERT 1000000 //convert sec (s) to microsec (us)
+
+//Set up test factors
+const long long unsigned int ALLOC_SIZE {137438953472}; //137GiB, total size of all allocations together
+const int SIZE {1<<28}; //268MiB, size of each allocation
+const int NUM_ALLOC {512}; //number of allocations for each round
```

Comment:
With my suggested changes, this declaration may be moved in to `setup_test` function.  No longer needed as a global.

Reply:
Done